index.js

```diff
@@ -56,15 +54,60 @@ const socketYmlSchema = {
   additionalProperties: false,
 }
 
+/**
+ * @typedef SocketYmlV1
+ * @property {string[]} [ignore]
+ * @property {{ [issueName: string]: boolean }} [issues]
+ * @property {boolean} [beta] unused v1 option
+ * @property {boolean} [enabled] enable/disable the Socket.dev GitHub app entirely
+ * @property {boolean} [projectReportsEnabled] enable/disable Github app project report checks
+ * @property {boolean} [pullRequestAlertsEnabled] enable/disable GitHub app pull request alert checks
+ */
+
+/** @type {import('ajv').JSONSchemaType<SocketYmlV1>} */
+const socketTmlSchemaV1 = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  type: 'object',
+  properties: {
+    ignore: {
+      type: 'array',
+      items: { type: 'string' },
+      nullable: true,
+    },
+    issues: {
+      type: 'object',
+      nullable: true,
+      required: [],
+      additionalProperties: { type: 'boolean' },
+    },
+    beta: { type: 'boolean', nullable: true, default: true },
+    enabled: { type: 'boolean', nullable: true, default: true },
+    projectReportsEnabled: { type: 'boolean', nullable: true, default: true },
+    pullRequestAlertsEnabled: { type: 'boolean', nullable: true, default: true },
+  },
+  minProperties: 1,
+  additionalProperties: false,
+}
+
 const ajv = new Ajv({
   allErrors: true,
   coerceTypes: 'array',
   logger: false,
   removeAdditional: 'failing',
+  useDefaults: true
 })
 
 const validate = ajv.compile(socketYmlSchema)
 
+const ajvV1 = new Ajv({
+  allErrors: true,
+  coerceTypes: 'array',
+  logger: false,
+  useDefaults: true
+})
```

Comment:
```suggestion
const ajvOptions = {
  allErrors: true,
  coerceTypes: 'array',
  logger: false,
  useDefaults: true
}

const ajv = new Ajv({
  ...ajvOptions,
  removeAdditional: 'failing',
})

const validate = ajv.compile(socketYmlSchema)

// We want to be strict and fail rather than removeAdditional when we parse a possible v1 config – only fallback to it when it actually matches well
const ajvV1 = new Ajv({
  ...ajvOptions
})
```

How about doing it like this to signify that's the difference?